docs/content/anchoredPosition.mdx

```diff
@@ -0,0 +1,96 @@
+---
+title: Anchored Position Behavior
```

Comment:
I tried to write most of this documentation exclusively for `getAnchoredPosition` since it should be relevant regardless of being in React-land or not. Therefore, I'd like to push back a bit on this suggestion, although maybe the answer is just to invert the order of things (describe the hook first, then below that details on how it works). I agree, in the future when it is upstreamed, the documentation will need to change to just focus on the hook, but at that point, this doc should be able to just be lifted and shifted.